setup.py

```diff
@@ -36,6 +36,7 @@
             packages=PACKAGES,
             package_data=PACKAGE_DATA,
             install_requires=REQUIRES,
+            extras_require={'docs': ['m2r2', 'sphinx']},
```

Comment:
Should there be a corresponding change in the readme that alerts potential docs users that they have to run something else to get the docs to install too?

Edit: I'm guessing that'll go in the contributing doc which is still in the works